generators/new-plugin/index.js

```diff
@@ -0,0 +1,103 @@
+var generators = require('yeoman-generator');
+var mkdirp = require('mkdirp');
+var rimraf = require( 'rimraf' );
+var fs = require('fs');
+var replace = require('replace');
+var merge = require('merge');
+
+module.exports = generators.Base.extend({
+
+  initializing: function () {
+    this.THEME_FOLDER = './wp-content/themes/';
+    this.PLUGIN_GIT_URI = 'git@github.com:moxie-leean/wp-plugin.git';
+  },
+
+  prompts: function() {
+    var done = this.async();
+
+    var questions = [
+      {
+        name: 'name',
+        message: 'The name of this project',
+        default: this.destinationRoot().split('/').pop(),
```

Comment:
This is the same as this one https://github.com/moxie-leean/generators/blob/new-plugin/generators/new-wp/index.js#L12 please create a shared function and apply the same function on both places.